blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/BaseTest.java

```diff
@@ -45,22 +42,9 @@
 public abstract class BaseTest {
     private static  final Logger LOGGER = LoggerFactory.getLogger(BaseTest.class);
 
-
-
     @Rule
-    public JenkinsRule j = new BaseTestJenkinsRule();
-
-    public static class BaseTestJenkinsRule extends JenkinsRule{
-        @Override
-        protected LoginService configureUserRealm() {
```

Comment:
so this was C&P everywhere and unused?

Reply:
unused? This is the only workaround that was possible to avoid jenkins-test-harness to not use jetty api 9.4.5.

Reply:
Basically configureUserRealm was overriden so that it uses version of jetty used by wiremock. This was as per discussed here, https://github.com/jenkinsci/jenkins-test-harness/pull/63/files#r141060896